lib/qesapdeployment.pm

```diff
@@ -558,7 +558,7 @@ sub qesap_execute {
 
     qesap_execute_conditional_retry(
     cmd => $qesap_script_cmd,
-    error_string => 'Fatal:',
+    error_list => 'Fatal:',
     logname => 'somefile.txt'
     [, verbose => 1s] );
 
```

Comment:
Could you please expand 
```
Execute qesap glue script commands. Check project documentation for available options:
    https://github.com/SUSE/qe-sap-deployment
Test only returns execution result, failure has to be handled by calling method.
```

Reply:
in particular about the return code (in case of retry). Does it die? Does it ....